### PR TITLE
.NET 6 conversion

### PR DESCRIPTION
**Pull request by jorgensigvardsson**
- Converted projects to .NET 6 (but left relevant projects at .NET Standard)
- Removed UAP library as it's not needed for our purposes
- Removed all `#if METRO` and `#elif METRO` directives
- Turned on nullable reference types

### CODE REVIEW

File: .idea/.idea.GraphX for .NET/.idea/.name
Comment (carl-andersson-at-westermo) [inline .idea/.idea.GraphX for .NET/.idea/.name:1]: These probably shouldn't be here / ignored.

File: .idea/.idea.GraphX for .NET/.idea/GitlabLint.xml
Comment (carl-andersson-at-westermo) [inline .idea/.idea.GraphX for .NET/.idea/GitlabLint.xml:1]: mentions westermos' gitlab, shouldn't be here

**Review by carl-andersson-at-westermo — CHANGES_REQUESTED** [review]
Some files that probably should be present are here.

File: .idea/.idea.GraphX for .NET/.idea/.name
Comment (jorgensigvardsson) [reply .idea/.idea.GraphX for .NET/.idea/.name:1]: Rider actually suggested to include them

File: .idea/.idea.GraphX for .NET/.idea/GitlabLint.xml
Comment (jorgensigvardsson) [reply .idea/.idea.GraphX for .NET/.idea/GitlabLint.xml:1]: Ooh! I'll fix it.

**Review by carl-andersson-at-westermo — APPROVED** [review]

**Pull request merged**